helm_deploy/calculate-journey-variable-payments/templates/_envs.tpl

```diff
@@ -118,4 +118,10 @@ env:
   - name: CRON_IMPORT_REPORTS
     value: "{{ .Values.env.CRON_IMPORT_REPORTS }}"
 
+  - name: SENTRY_DSN
```

Comment:
This might take a few attempts (hopefully not).  I think I have set up what is needed in k8s.